server/app/views/v1/grid_services/_grid_service.json.jbuilder

```diff
@@ -33,3 +33,15 @@ json.instances do
   json.running grid_service.containers.where('state.running' => true).count
 end
 json.hooks grid_service.hooks.as_json(only: [:name, :type, :cmd, :oneshot])
+if grid_service.health_check
+	json.health_check do
+		json.protocol grid_service.health_check.protocol
+		json.uri grid_service.health_check.uri
+		json.port grid_service.health_check.port
+		json.timeout grid_service.health_check.timeout
+		json.initial_delay grid_service.health_check.initial_delay
+		json.interval grid_service.health_check.interval
+	end
+	json.health_status grid_service.health_status
+end
+json.health_status grid_service.health_status if grid_service.health_check
```

Comment:
What kind of json response we will get if grid_service has health_check defined? Will there be duplicated health_status?